pynest/nest/tests/test_connect_arrays.py

```diff
@@ -80,15 +134,14 @@ def test_connect_arrays_no_delays(self):
         """Connecting NumPy arrays without specifying delays"""
         n = 10
         nest.Create('iaf_psc_alpha', n)
-        sources = np.arange(1, n+1, dtype=np.uint64)
-        targets = np.arange(1, n+1, dtype=np.uint64)
-        weights = np.ones(len(sources))
-        # delays = np.ones(len(sources))
-        syn_model = 'static_synapse'
+        sources = [i for i in range(1, n + 1)]
```

Comment:
It's better to use `list()` here.
```suggestion
        sources = list(range(1, n + 1))
```